packages/desktop-client/src/style/themes/dark.ts

```diff
@@ -10,7 +10,7 @@ export const pageBackgroundLineBottom = colorPalette.navy150;
 export const pageText = colorPalette.navy150;
 export const pageTextLight = colorPalette.navy300;
 export const pageTextSubdued = colorPalette.navy500;
-export const pageTextDark = colorPalette.navy800;
+export const pageTextDark = colorPalette.navy100;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏would this change require regeneration of the VRT screenshots?

Reply:
I don't think it's visible on any screenshots. It's being used for a loading animation that's not captured on any SS. Navy800 is not visible on darktheme background (hence the change).